tools/main-transpile-wrapper.js

```diff
@@ -0,0 +1,8 @@
+// This file exists because it is the file in the tool that is not automatically
+// transpiled by Babel.
+
+require('source-map-support').install();
```

Comment:
This function is already called in files.js.  And it probably can't actually be called more than once, sigh.  And our current version does Special Stuff instead of the default.  The default is to find the source maps based on  special comments in the source file itself; ours is to find them based on isopack metadata.  You'll probably want to change it so that where our implementation of `retrieveSourceMap` returns null, it instead invokes the default implementation (`sourceMapSupport.retrieveSourceMap`).

ie, you'll need a single function that knows how to find Meteor's source maps for files run with `files.runJavaScript` (ie, isopacket-loaded files, plugins) *and* source maps written by Babel